fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -192,21 +193,44 @@ public Response options() {
      * Retrieve the node profile
      *
      * @param rangeValue the range value
-     * @return triples for the specified node
+     * @return the binary or the triples for the specified node
      * @throws IOException if IO exception occurred
      */
     @GET
-    @Produces({TURTLE + ";qs=10", JSON_LD + ";qs=8",
-            N3, N3_ALT2, RDF_XML, NTRIPLES, APPLICATION_XML, TEXT_PLAIN, TURTLE_X,
-            TEXT_HTML, APPLICATION_XHTML_XML, "*/*"})
```

Comment:
I believe the issue is that this endpoint can produce _anything_. Hence the `@Produces({"*/*"})`. A better use of JAX-RS would be to have one endpoint that produces content-negotiated RDF (LDP-RS with explicit `Produces` values) and another that produces `Binary` (LDP-NR) resources where any content type is permitted. The difficulty with that approach is (AFAIK) that Fedora uses the same endpoint to produce both of these resource types.